app/src/main/java/com/example/applinting/kotlin/GlobalData.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.applinting.kotlin
+
+class GlobalData constructor(val name: String, val age: Int) {
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>62: The class or object GlobalData is empty. [EmptyClassBlock]